server/controllers/accesses.js

```diff
@@ -76,9 +76,9 @@ export async function create(req, res) {
 // Fetch operations using the backend. Note: client needs to get the operations
 // back.
 export async function fetchOperations(req, res) {
+    let access = req.preloaded.access;
```

Comment:
Can you move it back to where it was, please? If `req.preloaded` is undefined, this will silently swallow an error